dashboard/modules/train/train_head.py

```diff
@@ -44,8 +44,12 @@ async def get_train_runs(self, req: Request) -> Response:
         else:
             try:
                 train_runs = await stats_actor.get_all_train_runs.remote()
-                # TODO(aguo): Sort by created_at
-                details = TrainRunsResponse(train_runs=list(train_runs.values()))
+                train_runs = sorted(
+                    train_runs.values(),
+                    key=lambda run: run.start_time,
+                    reverse=True,
+                )
```

Comment:
nit: Add a comment just saying this is sorted in reverse chronological order.

Reply:
Done